CONTRIBUTING.md

```diff
@@ -8,6 +8,7 @@ Contributions are welcome from both inside and outside the institute.
 4. Create your notebook. It can be anywhere within the `notebooks` directory or any level of sub-directory, but each notebook should be in its *own* directory with no other notebooks or other files.
 5. Develop your notebook by adding and committing it (`git add path/to/my/notebook/notebook.ipynb`,  `git commit -m 'Added some feature'`), as many times as necessary to capture your development history.  Note that you should *never* commit a notebook with executed cells or commit large data files (see the guide).  Always "Clear Ouputs" on cells before adding/committing.
 6.  Add a ``requirements.txt`` file next to your notebook.  Each line should be a separate package following `pip` [requirements file conventions](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers), listing all the required packages for your notebook, including a "known good version" (e.g., if you wrote the notebook on numpy v1.14.0, the line should be ``numpy>=1.14.0``).
+    - **A note on supplemental files:** Do you need to add additional files to supplement your notebook that have one of the following suffixes:  ``.html``, ``.fits``, ``.coo``, ``.match``, ``.log``, ``.png``, ``.list``, ``.txt``, ``.cat``, ``.jpg``, ``.gif``, ``.zip``? If so, be sure to add that file name to the "Exceptions committed to repo" list within ``notebooks/notebooks/.gitignore``. For example, if you need to add ``diagram.jpg`` to your notebook ``DrizzlePac/sky_matching``, then add ``!DrizzlePac/sky_matching/diagram.jpg`` to the ``.gitignore`` list (don't forget the ``!``). Otherwise, git won't recognize your file. 
```

Comment:
Supplemental files should be discouraged so perhaps a message pointing towards the stye guide for supplemental files [data style guid](https://github.com/spacetelescope/style-guides/blob/master/guides/where-to-put-your-data.md)

And possibly a message stating they should only be included if they cannot be retrieved or are not accessible online (like a MAST download, etc.)